services/identity/MailClient.ts

```diff
@@ -38,3 +38,21 @@ class MailClient {
   }
 }
 export default MailClient
+
+const { NODE_ENV, POSTMAN_API_KEY } = process.env
+
+const IS_LOCAL_DEV = NODE_ENV === "LOCAL_DEV"
+
+if (!POSTMAN_API_KEY && !IS_LOCAL_DEV) {
+  throw new Error(
+    "Please ensure that you have set POSTMAN_API_KEY in your env vars and that you have sourced them!"
+  )
+}
+
+const mockMailer = {
+  sendMail: (email: string, subject: string, html: string) =>
+    logger.info(`Mock email sent to <${email}>, subject: ${subject}\n${html}`),
+} as MailClient
+export const mailer = IS_LOCAL_DEV
+  ? mockMailer
+  : new MailClient(POSTMAN_API_KEY!)
```

Comment:
Could I check why the initialisation was moved from index.ts into MailClient itself? I was thinking the index file would be a better place to handle initialisation, as this is consistent across our database currently

Reply:
I just spoke to @alexanderleegs about this, and we can't think of a better place to put this for now. I'll make a note to address this when we reorganize the service files.

Reply:
agreed with @rc-davis on the fact that `MailClient` doesn't seem to belong in `services/identity`! in view of time pressure, i think we're filing an issue to shift `MailClient` to `UtilServices` and rename it to `MailService`; this is fine as a band-aid fix **for now**

Reply:
I added this as a line item to this issue #461, which we could discuss on Monday.